lib/tts_isolate.dart

```diff
@@ -59,28 +61,59 @@ void onStart(ServiceInstance service) async {
 
   service.on('initSharedPreference').listen((event) async {
```

Comment:
Nice work, we should also switchMap on this one too. Let me know if you'd like an example.

Reply:
Yeah I might need an example for this one!

Reply:
I _might_ have gotten it. Let me know if the last commit does the trick?